### ISSUE TITLE
Fitness bug?

### ISSUE THREAD

**Issue by QuiescentRabbitt** [carykh/evolutionSteer#5]
Extremely odd bug, only encountered one time.

![image](https://user-images.githubusercontent.com/29581638/27350843-801ed7ae-55c9-11e7-9a45-91c15883a16b.png)

Pretty terrible evolution, only projectiles. However I had a spike of 9 fitness and found that midway through jump its chomps rapidly flickered ranging from 0.5 to 17.5 rapidly and then stopped at 8.9 and recorded that in the chart.

**Comment by COZITIME** [comment]
Yeah I have had the same thing occur. But mine spiked up to 90 on the chart. I have a video of it spazzing out, not sure what would cause this though. 


![image](https://user-images.githubusercontent.com/29688561/27515764-799ecde8-59ee-11e7-9cb2-a1ffd7ab3e14.png)

![image](https://user-images.githubusercontent.com/29688561/27515777-ea15e912-59ee-11e7-8739-6a540d070ee8.png)

**Comment by ghost** [comment]
![Capture](https://user-images.githubusercontent.com/52084152/90944175-d8f0df80-e3e2-11ea-9cef-37753ef88fc1.PNG)
I had a few giant spikes. Unfortunately, I was unable to few the footage of any of the spikes because of how many generations there were.
